docs/built-in-models/training.md

```diff
@@ -1,10 +1,25 @@
 # Encrypted training
 
-This document explains how to train [SGD Logistic Regression](../references/api/concrete.ml.sklearn.linear_model.md#class-sgdclassifier) on encrypted data. The [logistic regression training](../advanced_examples/LogisticRegressionTraining.ipynb) example shows this feature in action.
+This document explains how to train [SGD Logistic Regression](../references/api/concrete.ml.sklearn.linear_model.md#class-sgdclassifier) on encrypted data.
+
+Training on encrypted data is done through an FHE program that is generated by Concrete ML, based on the characteristics of the data that are given to the `fit` function. The FHE program associated to a `SGDClassifier` object, once it is fit on the encrypted data, is specific to the distribution and the dimensionality of that data.
+
+When deploying encrypted training services, developers need to consider the type of data that future users of their services will train on:
+
+- the distribution of the data should match to achieve good accuracy
+- the dimensionality of the data needs to match since the deployed FHE programs are compiled for a fixed number of dimensions.
```

Comment:
```suggestion
- The distribution of the data should match to achieve good accuracy
- The dimensionality of the data needs to match since the deployed FHE programs are compiled for a fixed number of dimensions.
```